src/components/valvonta-oikeellisuus/toimenpiteet.js

```diff
@@ -69,6 +68,18 @@ export const isAuditCaseToimenpideType = R.propSatisfies(
   'id'
 );
 
```

Comment:
tämä on luultavasti pakko tehdä backend:ssä

Reply:
Juu, näin päättelin. Eihän näistä koodeista sinällään haittaa ole, mutta efektiivisesti bäkkärimuutosten jäljiltä on kuollutta koodia. Tapetaan siis tämä review ja odotetaan bäkkäritoteutusta.